dask_saturn/__init__.py

```diff
@@ -1,4 +1,6 @@
 from .core import SaturnCluster
+from .core import describe_sizes
+from .core import list_sizes
```

Comment:
A minor comment for a minor PR, but this should probably just be a one liner :)

Reply:
done